src/libraries/Microsoft.PowerFx.Repl/PsuedoFunctions.cs

```diff
@@ -45,4 +47,43 @@ await repl.Output.WriteLineAsync(irText, OutputKind.Repl, cancel)
 
         public string Name => "IR";        
     }
+
+    /// <summary>
+    /// Print the suggestions of an expression. 
+    /// </summary>
+    public class SuggestionsPseudoFunction : IPseudoFunction
+    {
+        public async Task ExecuteAsync(CheckResult checkResult, string innerExp, PowerFxREPL repl, CancellationToken cancel)
+        {
+            if (innerExp.StartsWith("\"", StringComparison.OrdinalIgnoreCase) && innerExp.EndsWith("\"", StringComparison.OrdinalIgnoreCase))
+            {
+                innerExp = innerExp.Substring(1, innerExp.Length - 2);
+            }
+
```

Comment:
Why do we need to trim the `"` characters from the expression?

Reply:
The function receives a string literal from REPL: `"Abs(|"`.
We then need to trim the quotes to compile it like a proper expression `Abs(|`.

Reply:
Ah, I see, we need to pass the expression as a string, not as a "normal" expression... Could we do something like `Suggestions(Abs|)`? If not, then what you have is good then.

Reply:
Nevermind, it wouldn't work - as it would be a compile-time error. Please ignore my suggestion :-)